core/sql/src/main/java/org/trafodion/sql/udr/LmUDRObjMethodInvoke.java

```diff
@@ -68,7 +68,7 @@ static LmUDRObjMethodInvoke makeNewObj(
         return result;
     }
 
-    public static class ReturnInfo
+    public class ReturnInfo
```

Comment:
Just curious, I don't see a code change that makes this necessary, why are we changing the static class to an inner class? Wouldn't the static class be preferable, especially given that we pass it - without an object from the parent class - via JNI? My knowledge about this difference comes from https://docs.oracle.com/javase/tutorial/java/javaOO/nested.html, am not a real expert in Java nested classes.